azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -78,7 +78,13 @@ public ExecutableFlow fetchExecutableFlow(final int execId)
   @Override
   public Map<Integer, Pair<ExecutionReference, ExecutableFlow>> fetchUnfinishedFlows()
       throws ExecutorManagerException {
-    return new ConcurrentHashMap<>();
+    return this.activeFlows;
+  }
+
+  @Override
+  public Map<Integer, Pair<ExecutionReference, ExecutableFlow>> fetchUnfinishedExecutions()
+      throws ExecutorManagerException {
+    return this.activeFlows;
```

Comment:
unfinishedFlows might not be the same as activeFlows. Will it cause confusion here?